tutorials/en_us/JARS/OPTIMIZATION.md

```diff
@@ -0,0 +1,19 @@
+# Intro
+
+Here you will learn how to optimize you Minecraft server to get better performance.
+
+## Optimizing your .jar specific configurations to YOUR NEEDS
+
+Every .jar comes with its own set of configurations you can tweak to suit your needs, and keep in mind that, YOUR NEEDS. You probably wont find as effective the config of a minigames server in a survival server. You should read every config, understand it, and choose if you need to use it or not. This is extremely important in survival servers as some config values can ruin your mob spawning rates or destroy your players custom redstone circuits.
+
+Heres a [guide](https://www.spigotmc.org/threads/guide-server-optimization⚡.283181/) made by a well-known member of the community that can help you further in this optimization, it is limited to PaperMC and doesn't have optimized configs for Tuinity, Purpur or any other forks. Or join [this discord](https://discord.gg/yev2rN3eZH), they have an amazing discord bot that tells you how to optimize your server based on your timings report! Remember this is a general suggestion, it may not suit your needs.
```

Comment:
Heres --> Here is*

Split this into multipl;e lines for readability. 

Line 10 could be
`You can also join [this discord](link), which has an amazing Discord bot that assists you optimizing your server based on your timings report!`
(discord --> Discord, also general grammar fix)